src/components/articles/Article.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { colors, dimensions } from 'styles/variables';
+import ArticleHeader from 'components/articles/ArticleHeader';
+import ArticleDescription from 'components/articles/ArticleDescription';
+import ArticleImage from 'components/articles/ArticleImage';
+import ArticleContent from 'components/articles/ArticleContent';
+
+const Container = styled.div({
+  display: 'flex',
+  flexDirection: 'row',
+  padding: dimensions.paddings.md,
+  backgroundColor: colors.light.bg.primary,
+  borderRadius: 10,
+  '& + &': {
+    marginTop: dimensions.paddings.sm,
+  },
+});
+
+interface PostProps {
+  post: GatsbyTypes.MarkdownRemark;
+}
+
+const Article: React.FC<PostProps> = ({ post }) =>
+  post && post.frontmatter && post.fields ? (
+    <Container>
+      {post.frontmatter.image?.src ? (
+        <ArticleImage
+          file={post.frontmatter.image.src}
+          alt={post.frontmatter.image.alt}
+        />
+      ) : null}
```

Comment:
I recommand to use operator `&&` instead of ternary operator. How about you?

Reply:
Sound good to me
I will reflect it

Reply:
I'll switch it as resolved